acis_thermal_check/state_builder.py

```diff
@@ -263,6 +263,10 @@ def __init__(self, interrupt=False, backstop_file=None, nlet_file=None,
             self.tstart = rev_bs_cmds[0]['time']
             self.tstop = rev_bs_cmds[-1]['time']
 
+            # Initialize the end time attribute for event searches within the BSC object
+            # At the beginning, it will be the time of the last command in the Review Load
+            self.BSC.end_event_time = rev_bs_cmds[-1]['time']
```

Comment:
Ideally we'd have this as a setting function (e.g. `self.BSC.set_end_event_time(rev_bs_cmds[-1]['time'])` instead of just setting the attribute, but we can worry about that later.

Reply:
Or make it a property.